k6/vai/create_vai_resources.js

```diff
@@ -0,0 +1,109 @@
+import encoding from 'k6/encoding';
+import exec from 'k6/execution';
+import { createConfigMaps, createSecrets, createDeployments } from '../generic/generic_utils.js';
+import {check, fail, sleep} from 'k6';
+import * as k8s from '../generic/k8s.js'
+import http from 'k6/http';
+
+
+// Parameters
+const namespace = "scalability-test"
+const secretData = encoding.b64encode("a".repeat(10*1024))
+const confgMapData = encoding.b64encode("a".repeat(10*1024))
+const secretCount = Number(__ENV.SECRET_COUNT)
+const configMapCount = Number(__ENV.CONFIGMAP_COUNT)
+const deploymentCount =Number(__ENV.DEPLOYMENT_COUNT)
+const vus = 10
+
+// Option setting
+const kubeconfig = k8s.kubeconfig(__ENV.KUBECONFIG, __ENV.CONTEXT)
+const baseUrl = kubeconfig["url"].replace(":6443", "")
+const username = __ENV.USERNAME
+const password = __ENV.PASSWORD
+
+export const options = {
+    insecureSkipTLSVerify: true,
+    tlsAuth: [
+        {
+            cert: kubeconfig["cert"],
+            key: kubeconfig["key"],
+        },
+    ],
+
+    setupTimeout: '8h',
+
+    scenarios: {
+        createVaiResourcesSecrets: {
+            executor: 'shared-iterations',
+            exec: 'createVaiResourcesSecrets',
+            vus: vus,
+            iterations: secretCount,
+            maxDuration: '1h',
+        },
+        createVaiResourcesConfigMaps: {
+            executor: 'shared-iterations',
+            exec: 'createVaiResourcesConfigMaps',
+            vus: vus,
+            iterations: configMapCount,
+            maxDuration: '1h',
+        },
+        createVaiResourcesDeployments: {
+            executor: 'shared-iterations',
+            exec: 'createVaiResourcesDeployments',
+            vus: vus,
+            iterations: deploymentCount,
+            maxDuration: '1h',
+        },
+    },
+    thresholds: {
+        checks: ['rate>0.99']
```

Comment:
Can we update the thresholds to the following as our baseline for this test?
```suggestion
http_req_failed: ['rate<=0.01'], // http errors should be less than 1%
http_req_duration: ['p(99)<=500'], // 99% of requests should be below 500ms
checks: ['rate>0.99'], // the rate of successful checks should be higher than 99%
```